src/widgets/histogram/chart.js

```diff
@@ -96,7 +96,7 @@ module.exports = cdb.core.View.extend({
 
     this.formatter = formatter.formatNumber;
     if (this._isDateTimeSeries()) {
-      this.formatter = formatter.timestampFactory(this._dataviewModel.get('aggregation'));
+      this.formatter = formatter.timestampFactory(this._dataviewModel.get('aggregation'), this._dataviewModel.get('offset'));
```

Comment:
If everytime the timezone changes, we re-render the chart, we can pass that value to this function and then saving the `get('local_timezone')` in every call, exposing the same call to both numeric and date values.

Indeed, when we call `this.formatter` for numeric values, the parameter `local_timezone` has no sense.

But perhaps, it cannot be done if `initialize` is not called every time the timezone changes.